RemoveChatButtons.plugin.js

```diff
@@ -186,6 +186,14 @@ module.exports = (() => {
                         note: 'Removes the seasonal "Discord\'s Birthday" tab button from the DM list.',
                         value: true,
                     },
+                    {
+                        type: 'switch',
+                        id: 'discordShopTab',
+                        name: 'Remove Discord\'s Shop Tab',
+                        note: 'Removes the Discord Shop tab button from the DM list.',
+                        value: false,
+
+                    }
```

Comment:
There should be a trailing comma here to maintain the file style (and keep future additions clean).